views/Feed/Feed.jsx

```diff
@@ -140,18 +140,14 @@ export default class FeedView extends React.Component {
   }
 
   updatePost = async (postId, data, type) => {
-
-    const {
-      navigation,
-    } = this.props;
-
     if (type === 'toggleLike') {
-      this.setState({
-        posts: this.state.posts.map(post => {
-          if (post._id === postId) return data;
-          return post;
+      return ApiClient.post(`/posts/${postId}/like`, {'addLike': data.isLiked}, {authorized: true})
+        .then(() => {
+          this.updateState('updatePost', postId);
```

Comment:
I don't think this is doing anything.. The 'updatePost' logic expects a post object with data to be updated, since `postId._id` is undefined it looks like this will just make the component do a redundant state update. Maybe pass `data`?